src/stateToHTML.js

```diff
@@ -153,19 +157,23 @@ class MarkupGenerator {
   options: Options;
   inlineStyles: StyleMap;
   styleOrder: Array<string>;
+  blockTags: BlockTags;
+  blockWrapperTag: BlockWrapperTag;
 
-  constructor(contentState: ContentState, options: ?Options) {
-    if (options == null) {
-      options = {};
-    }
-    this.contentState = contentState;
-    this.options = options;
-    let [inlineStyles, styleOrder] = combineOrderedStyles(
-      options.inlineStyles,
-      [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER],
-    );
-    this.inlineStyles = inlineStyles;
-    this.styleOrder = styleOrder;
+  constructor(contentState: ContentState, options: ?Options) {
```

Comment:
Sorry, not sure why this entire block changed, maybe spacing or new line related. `this.blockTags` and `this.blockWrapperTag` is the only I added to the constructor